apps/app/src/styles/variables.css

```diff
@@ -1,7 +1,43 @@
 :root {
 	--primary: theme("colors.violet.700");
+
+	--junior-light-color: #edf3f8;
+	--junior-main-color: #499dff;
+	--junior-main-color-dark: #1682ff;
+	--junior-main-color-darker: #0068e2;
+	--junior-main-color-light: #7cb8ff;
+
+	--mid-light-color: #edf8f2;
+	--mid-main-color: #27be31;
+	--mid-main-color-dark: #1e9426;
+	--mid-main-color-darker: #16691b;
+	--mid-main-color-light: #40d84a;
+
+	--senior-light-color: #fffbf0;
+	--senior-main-color: #ffb90b;
+	--senior-main-color-dark: #d79900;
+	--senior-main-color-darker: #a47500;
+	--senior-main-color-light: #ffc83e;
```

Comment:
Jeśli zamienimy te kolory na oklch to może okaże się, że można je wygenerować - tzn w oklch z jednego koloru łatwo stworzyć wersje light, dark i darker.